test-plan/eventing/channel-lifecycle-conformance.md

```diff
@@ -0,0 +1,229 @@
+# Channel Lifecycle 
+
+From: https://github.com/knative/specs/blob/main/specs/eventing/control-plane.md#channel-lifecycle
+
+
+From the Spec: 
+
+>> A Channel represents an Addressable endpoint (i.e. it MUST have a status.address.url field) which can receive, store, and forward events to multiple recipients (Subscriptions).
+
+>> Subscriptions MUST be associated with a Channel based on the spec.channel field on the Subscription; it is expected that the controller for a Channel will also control the associated Subscriptions.
+
+>> When the Channel's Ready condition is true, the Channel MUST provide a status.address.url which accepts all valid CloudEvents and MUST attempt to forward the received events to each associated Subscription whose Ready condition is true. As described in the Subscription Lifecycle section, a Channel MAY forward events to an associated Subscription which does not currently have a true Ready condition, including events received by the Channel before the Subscription was created.
+
+>> When a Channel is created, its spec.channelTemplate field MAY be populated to indicate which of several possible Channel implementations to use. It is RECOMMENDED to default the spec.channelTemplate field on creation if it is unpopulated. Once created, the spec.channelTemplate field MUST be immutable; the Channel MUST be deleted and re-created to change the spec.channelTemplate. This pattern is chosen to make it clear that changing spec.channelTemplate is not an atomic operation and that any implementation would be likely to result in event loss during the transition.
+
+
+
+# Testing Channel Lifecycle Conformance: 
+
+We are going to be testing the previous paragraphs coming from the Knative Eventing Spec. To do this we will be creating a Channel, checking its immutable properties, checking its Ready status and then creating a Subscription that links to it by making a reference. We will also checking the Subscription status, as it depends on the Channel to be ready to work correctly. We will be also checking that the Channel is addressable by looking at the status conditions fields. Because this is a Control Plane test, we are not going to be sending Events to the components. 
```

Comment:
After reviewing this file I noticed that events are being sent as part of this conformance tests.
Please, update the paragraph above to remove the text that indicates otherwise.